internal/opentelemetry/metrics.go

```diff
@@ -58,7 +58,7 @@ var (
 		metric.WithDescription("Total number of resolvers completed"),
 		metric.WithUnit(unit.Dimensionless))
 
-	resDuration = metric.Must(meter).NewInt64ValueRecorder("resolver.duration.ms",
+	resDuration = metric.Must(meter).NewInt64Histogram("resolver.duration.ms",
```

Comment:
@negz did you see a change in behavior switching to `NewInt64Histogram`? 

Based on:
- https://github.com/open-telemetry/opentelemetry-go/issues/1280
- and the linked issue in 1280 https://github.com/open-telemetry/opentelemetry-go/issues/689#issuecomment-622137029

```
OpenTelemetry Metrics is explicitly moving away from instruments named for the action they 
perform, and moving toward instruments named for how they are used. There is no Histogram 
instrument, which is part of the reason buckets can't be configured at the instrument level. 
In the forthcoming 0.4 metrics specification, there will be two instruments that will translate 
into Prometheus histograms by default, those are going to be named ValueRecorder and 
ValueObserver
```

It seems like we should be seeing histograms from using `ValueRecorder`.

Reply:
@tnthornton I didn't test for a behaviour change, but as far as I can tell this was just a rename in upstream. See for example https://github.com/open-telemetry/opentelemetry-go/pull/2202 and specifically https://github.com/open-telemetry/opentelemetry-go/pull/2202/files#diff-06b5bbc47352183e5b82d7dfca06cafd665876b31dc9672bc7a99fa9ea4fdba7L78.

The otel metrics API has been a bit of a wild ride, but it seems like they settled on having a "Histogram" type after all.

Reply:
Sounds good to me. I just wanted to make sure we weren't going backwards 👍